src/MergeContext.js

```diff
@@ -497,19 +497,26 @@ class PullRequest {
         return null;
     }
 
+    // whether the given GitHub user is a core developer
+    _coreDeveloper(role, userLogin, userId) {
+        const coreId = Config.coreDeveloperIds().get(userLogin);
+        if (!coreId)
+            return false;
+        this._log(`found core ${role}: ${userLogin}={userId}`);
```

Comment:
Just polishing to better blend with the actual `role` parameter values:

```suggestion
        this._log(`${role} is a core developer: ${userLogin}={userId}`);
```

Reply:
Somehow the old line did not go away when the replacement was added. We can fix it later, I guess.